configure.d/1_blk_mq.conf

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Copyright(c) 2012-2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+. $(dirname $3)/conf_framework
+
+check() {
+	cur_name=$(basename $2)
+	config_file_path=$1
+	if compile_module $cur_name "struct blk_mq_ops ops; ops.map_queue;" "linux/blk-mq.h"
+	then
+		echo $cur_name "1" >> $config_file_path
+	else
+		echo $cur_name "2" >> $config_file_path
+    fi
+}
+
+apply() {
+    case "$1" in
+    "1")
+		add_define "CAS_BLK_MQ_OPS_MAP_QUEUE" ;;
+    "2")
+    		return 0 ;;
```

Comment:
just `return 0;` in header file? does it even compile?

Reply:
This is just return code from bash function, it does not get printed to output file.